### PR TITLE
release nodemailer transport resources

### PR DESCRIPTION
**Pull request by AlvaroVega**
Maybe some manual restarts could be save with this PR.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+- Release transport resources of nodemailer after send email
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Is there any issue number associated? (I guess no, but to confirm...)

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: No issue was opened

Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: release transport resources of nodemailer after send email
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**